helm/generic-node/templates/generic-node.yaml

```diff
@@ -0,0 +1,199 @@
+        - "-upload-blocks-to-gcloud"
+        - "true"
+        {{- end }}
+        env:
+        - name: RAYON_NUM_THREADS
+          value: "6"
+        - name: DAEMON_REST_PORT
+          value: {{ .Values.ports.graphql | quote}}
+        - name: DAEMON_CLIENT_PORT
+          value: {{ .Values.ports.client | quote}}
+        - name: DAEMON_METRICS_PORT
+          value: {{ .Values.ports.metrics | quote}}
+        - name: MINA_CLIENT_TRUSTLIST
+          value: "10.0.0.0/8"
+        - name: CODA_CLIENT_TRUSTLIST
+          value: "10.0.0.0/8"
+        - name: CONNECT_PRIVATE_IPS
+          value: "true"
+        - name: GCLOUD_KEYFILE
+          value: "/gcloud/keyfile.json"
+        - name: NETWORK_NAME
+          value: {{ .Values.testnetName}}
+        - name: GCLOUD_BLOCK_UPLOAD_BUCKET
+          value: "features_network_block_data"
+        - name: DAEMON_EXTERNAL_PORT
+          value: {{.Values.ports.p2p | quote}}
+        ports:
+        - name: client-port
+          protocol: TCP
+          containerPort: {{.Values.ports.client}}
+        - name: graphql-port
+          protocol: TCP
+          containerPort: {{.Values.ports.graphql}}
+        - name: metrics-port
+          protocol: TCP
+          containerPort: {{.Values.ports.metrics}}
+        - name: external-port
+          protocol: TCP
+          containerPort: {{.Values.ports.p2p}}
+          hostPort: {{.Values.ports.external}}
+        volumeMounts:
+        {{- if .Values.features.uploadBlocksToGCloud.enabled}}
+        - name: gcloud-keyfile
+          mountPath: "/gcloud/"
+        {{- end }}
+        {{- if .Values.features.runtimeConfig.enabled}}
+        - name: daemon-config
+          mountPath: "/config/"
+        {{- end }} 
+        {{- if .Values.libp2p.staticKeypair.enabled}}
+        - name: destination-libp2p
+          mountPath: /root/libp2p-keys
+        {{- end }} 
+      volumes:
+      {{- if .Values.features.runtimeConfig.enabled}}
+      - name: daemon-config
+        configMap:
+          name: seed-daemon-config
+      {{- end }}
+      {{- if .Values.features.uploadBlocksToGCloud.enabled}}
+      - name: gcloud-keyfile
+        secret:
+          secretName: gcloud-keyfile
+          defaultMode: 256
+          items:
+          - key: keyfile
+            path: keyfile.json
+      {{- end }}
+      {{- if .Values.libp2p.staticKeypair.enabled}}
+      - name: source-libp2p-keys
+        secret:
+          secretName: {{ .Values.libp2p.staticKeypair.secretName }}
+          defaultMode: 448
+          items:
+          - key: key
+            path: key
+          - key: pub
+            path: key.pub
+      {{- end }}
+      - name: destination-libp2p
+        emptyDir: {}
+{{- include "nodeSelector.preemptible" .Values | indent 6 }}
+---
+{{- if .Values.features.exposeGraphql.enabled}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.name}}-graphql
+spec:
+  {{- if .Values.libp2p.loadBalancer.enabled}}
```

Comment:
FYI, I was wrong when I added this here, lines 185 to 189 here actually belong in `helm/generic-node/templates/generic-node-service.yaml`. This feature `{{- if .Values.features.exposeGraphql.enabled}}` and everything after line 178 can be removed